ansible/roles/dev-desktop/files/scripts/init_git.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3
+import requests
+import re
+import subprocess
+import getpass
+
+for cfg in ["email", "name"]:
+    if subprocess.run(["git", "config", "--global", f"user.{cfg}"], stdout=subprocess.DEVNULL).returncode == 0:
+        print("git username and email already configured")
+        exit(0)
+
+team = requests.get("https://raw.githubusercontent.com/rust-lang/team/gh-pages/v1/teams/cloud-compute.json")
```

Comment:
This should use the endpoint for the permission, the `cloud-compute` team is temporary:

https://team-api.infra.rust-lang.org/v1/permissions/dev_desktop.json